modular_zubbers/maps/biodome/pride_flags.dm

```diff
@@ -3,41 +3,113 @@
 	. = ..()
 	icon = initial(fake_type.icon)
 
+
+// HORIZONTAL
+
 /obj/structure/sign/flag/pride
-	name = "flag of Gay Pride"
-	desc = "The flag of Gay Pride. Hang that rainbow up with pride!"
+	name = "coder pride flag"
+	desc = "You probably shouldn't be seeing this. Yell at the coders about it."
```

Comment:
If you add the purple black checker texture to this one I will love you

Reply:
Coders need to install Counter Strike: Source